Jula

astridronn opened the pull request:
Jula rt templates done, creating pull request to edit the templates after a final code review.

astridronn committed:
Tiny update on 300x600

Max width on "productName" updated on 300x600 only

astridronn committed:
Merge branch 'Jula' of https://github.com/SE-GroupM/dco-files into Jula

Joaost commented:
CODE REVIEW JULA

  Skapa en pullrequest till develop så kan vi lägga in kommentarerna likt en riktig code review.
  
  1. Ta bort gamla kommentarer som kommer från "standard"-mallen. Exv rad 19
  get content placeholders from Choreograph Adsets, och rad 26-29

  2. Skapa en lokal variabel för content så att adset-datan bara anropas en gång, för att minimera antalet anrop mot adsetet.
  exv: var local_content = conent;
  och sedan använda local_content överallt istället.

  3. Byt då ut content även på rad 49, även byt namn på lokala collection variabeln så den inte heter exakt likadant som collectionet, exv:
  var product_collection = content.product_collection.value; ->  local_product_collection = local_content.product_collection.value;
  uppdatera då på alla ställen där product_collection används.
  
  4. Hitta ett passande namn på timeLine:en. för någon ny eller någon hos CC som ska felsöka koden kommer anta att det finns en TimeLine 1 (t1)
  då timeline:en heter t2. Så byt till main_timeline eller nåt passande.

  5. Click eventet för worldclick används inte. Ta bort den då. Bra och ha ifall man skulle vilja ha ett annat click på exv loggan. Men logiken här
  så kommer vi bara använda ett klick till produkt.

aslarsson committed:
Jula all formats

Started code clean up (all formats) and css of price elements according to guidelines on 980x600, 980x300, 980x240 (not done)

aslarsson committed:
Jula all formats

CSS adjusted according to guidelines (Price elements, colors, font sizes) and code clean up according to code review

aslarsson committed:
Jula price element

Added large styling 250x600

aslarsson committed:
Updated all Jula formats

300x250, 320x400 has medium-CSS for price elements, rest has large-CSS. Changed from slice() to replace() on product savings, to be able to style ".-" according to giudelines

aslarsson committed:
CSS update for Windows

Price elements are adjusted to not overlap in Windows

aslarsson committed:
980x240 new design

Started to implement new design on elements

aslarsson committed:
980x240 Jula new design

Removed main copy and cta button. Added variables to control color of product box elements. Added lowest price last 30 days text. Aligned product info to be centered in its container.

aslarsson committed:
980x240 Jula new design

Added so sup-class is added on price with price types sale and julaclub

aslarsson committed:
Jula new design

Updated 980x300, 980x600, 640x320, 300x600 and 250x600 according to new guidelines

aslarsson committed:
Jula all formats

Changed direction of animation to follow Bannerflow creatives

aslarsson committed:
Jula all formats

Fix for JulaClub price element - latest update: Line-height of span style element

aslarsson committed:
Jula all formats

Edited productPriceType to check value correct, due to updates in pixel

Joaost committed:
Added new animation code for 250x600 and 300x250 formats. Since we needed to change in the HTML structure and move out productInfo (containing price element and product name), I also updated the CSS for these elements
